apps/src/StudioApp.js

```diff
@@ -313,12 +323,12 @@ StudioApp.prototype.init = function(config) {
 
   var aniGifPreview = document.getElementById('ani-gif-preview');
   if (config.level.aniGifURL) {
-    aniGifPreview.style.backgroundImage = "url('" + config.level.aniGifURL + "')";
+    aniGifPreview.style.backgroundImage = 'url("' + config.level.aniGifURL + '")';
```

Comment:
You'll see a lot of these; some "quote-consistency" linting I turned off, corrected for, then turned off again.